domain/BonusNumber.java

```diff
@@ -0,0 +1,39 @@
+package PACKAGE_NAME.domain;
+
+import java.util.Objects;
+import java.util.Set;
+
+public class BonusNumber {
+
+    private final int bonusNumber;
+    private static final int MIN_VALUE = 1;
+    private static final int MAX_VALUE = 45;
```

Comment:
```suggestion
    private static final int MIN_VALUE = 1;
    private static final int MAX_VALUE = 45;
    
    private final int bonusNumber;
```

상수가 더 위에 위치하는 것이 컨벤션이고
상태와 상수는 띄어주는 것이 가독성에 더 좋습니다.